syntheseus/search/utils/misc.py

```diff
@@ -0,0 +1,2 @@
+def lookup_by_name(module, name):
+    return module.__dict__[name]

```

Comment:
Nit: this is effectively just an alias for `module.__dict__`. Any reason you made this rather than just writing `module.__dict__`?

Reply:
No strong reason; `module.__dict__` just looked a bit like a cryptic hack, and I thought it's maybe marginally nicer to have it wrapped in a utility function instead of repeating it...